File: src/python/1.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        index={}
        for i,num in enumerate(nums):
            if target - num in index:
                return [index[target - num] + 1,i+1]
            else:
                index[num] = i

File: src/cpp/1.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        //num,index
        unordered_map<int, int> map;
        vector<int> result;
        for(int i = 0; i< nums.size();++i)
        {
            if(map.count(target - nums[i]) )
            {
                result.push_back(map[target - nums[i]] + 1);
                result.push_back(i+1);
                return result;
            }
            else
            {
                map[nums[i]] = i;
            }
        }
    }
};
